.circleci/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM ubuntu
-MAINTAINER Michael Bartling "michael.bartling15@gmail.com"
+LABEL maintainer="Michael Bartling <michael.bartling15@gmail.com>"
 
-RUN apt-get update
-RUN apt-get install -y \
+RUN apt-get update && \
+    apt-get install -y \
```

Comment:
This will not cache the apt update. This actually slows down the build time.

Reply:
I'll revert it.
Just wondering why the apt update is not cached?

Reply:
Each line with a `RUN` command gets fake cached for changes and docker uses a set of rules to decide whether to run the line or just re use an existing build layer. There is a rule for `apt-get update` but if you have an install line after it then it doesn't always use the cache. It's literally only an issue on the ubuntu builds. I usually try to stick with alpine, but glibc is a bitch on it

Reply:
@mbartling 
I check the docker documentation.
What you suggests seem to against its best practice over here:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Weird, I remember it saying to not do that a few months ago. Might have changed. kk keep it in

Reply:
@dboyliao Let's make this a separate PR for better tracking though